tests/_support/AcceptanceTester.php

```diff
@@ -64,4 +64,21 @@ public function bootstrapWith( callable $function, string $file = null, bool $ro
 
 		return $rollback_callback;
 	}
+
+	/**
+	 * Run ElasticPress indexing command.
+	 *
+	 * WPBrowser DB management commands to not trigger WordPress actions, and hence Elastic does not keep track
+	 * of content updates and index them. This command makes sure Elastic index is updated to reflect the changes.
+	 *
+	 * @param string $options Additional arguments to append to the indexing command.
+	 * @return void
+	 */
+	public function reindexElastic( string $options = '' ) {
```

Comment:
Need to decide what this helper method is to be called and keep it consistent with the docs. `reindexElastic` versus `reindexContent` currently.

Again my preference is to be explicit about Elasticsearch versus Elastic so I like the `reindexContent` version but I'm not too concerned either way for the method name.

Reply:
🤦 Yeah, I think I prefer `reindexContent` too, I forgot to update it here.

Reply:
```suggestion
	public function reindexContent( string $options = '' ) {
```